src/test/a11y/a11y.ts

```diff
@@ -53,23 +46,27 @@ function expectNoErrors(messages: PallyIssue[]): void {
   }
 }
 
-function testAccessibility(url: string): void {
+function testAccessibilityWithActions(url: string, actions: string[]): void {
   describe(`Page ${url}`, () => {
     test('should have no accessibility errors', done => {
       ensurePageCallWillSucceed(url)
-        .then(() => runPally(agent.get(url).url))
+        .then(() => runPallyWith(url, actions))
         .then((result: Pa11yResult) => {
           expectNoErrors(result.issues);
           done();
         })
         .catch((err: Error) => done(err));
     });
   });
+
+}
+
+function testAccessibility(url: string): void {
+  testAccessibilityWithActions(url, []);
 }
 
 describe('Accessibility', () => {
```

Comment:
I know this follows the same style as the FaCT project but I am not keen on the way it is doing the tests within a function. I prefer to separate the tests and the test logic. I think you can use parameterized tests with Jest like below:

const parameters = [
    { url: '/.' },
     // Other URLs go here
  ];

parameters.forEach((parameter) => {
    it('Page \'' + parameter.url + '\' should ave no accessibility errors', done => {
        testAccessibilityWithActions(parameter.url, []);
    });
}

Reply:
We discussed about it, decided to keep it as it is for now